scikeras/_saving_utils.py

```diff
@@ -0,0 +1,137 @@
+import os
+import tempfile
+import zipfile
+
+from io import BytesIO
+from types import MethodType
+from uuid import uuid4 as uuid
+
+import numpy as np
+
+from tensorflow import io as tf_io
+from tensorflow import keras
+from tensorflow.keras.models import load_model
+
+
+def _get_temp_folder():
+    if os.name == "nt":
+        # the RAM-based filesystem is not fully supported on
+        # Windows yet, we save to a temp folder on disk instead
+        return tempfile.mkdtemp()
+    else:
+        return f"ram://{tempfile.mkdtemp()}"
```

Comment:
@stsievert this is the "secret sauce" to support the ram based filsystem on *nix and disk based on windows